spark/processing/3.4/py3/docker/py39/Dockerfile.cpu

```diff
@@ -1,12 +1,14 @@
-FROM 137112412989.dkr.ecr.us-west-2.amazonaws.com/amazonlinux:2
+FROM public.ecr.aws/amazonlinux/amazonlinux:2
```

Comment:
ask to confirm that access public ecr will not create security issue on our side?

Reply:
we should have always used public.ecr.aws/amazonlinux/amazonlinux:2 as it has the latest publicly released AL2. EMR also uses the same: https://code.amazon.com/packages/AwsChicagoImageBuildResources/blobs/555b5468092d89cd12d38d577992b334481b3920/--/src/main/resources/release-build/spark3/vanilla/emr-6.15.0/Dockerfile?#L1